src/components/inputField/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './PasswordField/PasswordField'
+export * from './TextField/TextField'
```

Comment:
This file isn't necessary. Instead, in the `index.ts` file located at the top level of the `components` folder, you can import directly from your component files. You can find an example of this in that same `index.ts` file for the `Navbar` component.